ROADMAP.md

```diff
@@ -4,32 +4,37 @@ This document defines a high level roadmap for Karmada development and upcoming
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep karmada progressing and shipping exciting and valuable features.
 
+## 2022 H1
+- Multi-cluster HA scheduling policy
+  * spread by region
+  * spread by zone
+  * spread by provider
+- Multi-cluster Ingress
+- Multi-cluster HPA(Horizontal Pod Autoscaling)
+- Federated Resource Quota
+- API Reference
+- [Karmada website](https://karmada.io/) refactor
+- Policy-based governance, risk, and compliance
+- Multi-cluster DNS (Cluster identity)
+- Multi-cluster Global Search
+- Scheduling re-balance
 
-## 2021 Q1
-- K8s native API support for multi-cluster applications
-- Multi-cluster application status aggregation
-- Per-cluster configuration customization for applications
-- Cluster Affinity Scheduling Policy
-- Multi-cluster HA scheduling policy: spread by cluster
-- Cluster registry; Support register existing clusters
-
-## 2021 Q2
-- Multi-cluster application automatic migration (DR)
-- Dynamically split replicas of applications based on multiple dimensions
-- Taint Toleration for clusters
-- Multiple sync mode: Push, Pull
-- Cluster lifecycle management
-
-## 2021 Q3
-- Multi-cluster service discovery
-- Load ingress support across clusters
-- Multi-cluster horizontal auto-scaling for applications
-- Multi-cluster resource re-balancing
-- More Multi-cluster HA scheduling policy: spread by region, zone, provider
-- Aggregated K8s API endpoint
-
-## 2021 Q4
-- CNI adoption with multiple underlying network setup
-- Multi-cluster Service Mesh Support
-- Multi-cluster monitoring & Logging
-- GitOps support
+## 2022 H2
+- Karmada Dashboard - alpha release
+- Karmada Scalability Baseline(performance report)
+- Cluster Addon
+- Helm chart propagation
+- Multi-cluster Events governance
+- Multi-cluster Operator Specification
+- Multi-cluster Application
+- Multi-cluster monitor
+- Multi-cluster Log governance
+- Multi-cluster storage management
+- Multi-cluster RBAC
+- Multi-cluster Network
+- Data migration across clusters
```

Comment:
`Data` means pod volume? What's the different between `Multi-cluster storage`?

Reply:
No, you can refer to https://github.com/karmada-io/karmada/discussions/1097 for migration thing, something like that.

And `Multi-cluster storage` is about Volume, PVC, etc.